packages/schematics/angular/utility/ast-utils.ts

```diff
@@ -344,6 +344,28 @@ export function getFirstNgModuleName(source: ts.SourceFile): string|undefined {
   return moduleClass.name.text;
 }
 
+export function getMetadataField(
```

Comment:
I believe this can be simplified a bit

```ts
export function getMetadataField(
  node: ts.ObjectLiteralExpression,
  metadataField: string,
): ts.ObjectLiteralElement[] {
  return node.properties
    .filter(prop => ts.isPropertyAssignment(prop))
    // Filter out every fields that's not "metadataField". Also handles string literals
    // (but not expressions).
    .filter(({ name }) => {
      return (ts.isIdentifier(name) || ts.isStringLiteral(name))
        && name.text === metadataField;
    });
}


```

Reply:
Sounds good. By the way, this is more like a question -- is there any valid reason for using `text` property over `getText()`? I noticed that in one of the functions that I introduced, using `text` breaks the tests because it's `undefined`, while `getText` returns `''` (ref: `typescript.js:96792`). Maybe using `getText` is safer since it always returns a string (not that it matters in the current change)?

Reply:
When using to compare, returning `undefined` is perfectly fine. using `text` directly. Is slightly more performant.